spec/models/parsed_raw_event_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe ParsedRawEvent do
+  let(:raw_event) { create(:raw_event) }
+  let(:parsed_raw_event) { described_class.new(raw_event) }
+
+  describe '.meta' do
+    subject { parsed_raw_event.meta }
+
+    %i(controller action url start finish duration).each do |key|
+      it "should include key '#{key}'" do
+        is_expected.to have_key key
+      end
+    end
+  end
+
+  describe '.details' do
+    subject { parsed_raw_event.details }
+
+    it "should be Array" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.